livekit-agents/livekit/agents/voice/room_io/_output.py

```diff
@@ -244,7 +244,7 @@ def _on_track_published(
     ) -> None:
         if (
             not self._is_local_proxy_participant(participant)
-            or track.source != rtc.TrackSource.SOURCE_MICROPHONE
+            or track.source not in [rtc.TrackSource.SOURCE_MICROPHONE, rtc.TrackSource.SOURCE_SCREENSHARE_AUDIO]
```

Comment:
The audio track published by RoomIO is always  `SOURCE_MICROPHONE`

```suggestion
            or track.source != rtc.TrackSource.SOURCE_MICROPHONE
```